change/@fluentui-react-conformance-1461c075-e920-4b98-ad27-2adbf47103e0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "add 'consistent-callback-args' test for event handler arguments",
+  "packageName": "@fluentui/react-conformance",
+  "email": "olfedias@microsoft.com",
+  "dependentChangeType": "patch"
```

```suggestion
  "dependentChangeType": "none"
```

Let's avoid explosion release 😏